ostree_upload_server/flatpak_importer.py

```diff
@@ -23,32 +25,73 @@
 
 
 class FlatpakImporter(object):
+    CONFIG_PATHS = [ '/etc/ostree/flatpak-import.conf',
+                     os.path.expanduser('~/.config/ostree/flatpak-import.conf'),
+                    'flatpak-import.conf' ]
+
+    METADATA_KEYS = [ 'ref',
+                      'flatpak',
+                      'origin',
+                      'runtime-repo',
+                      'metadata',
+                      'gpg-keys' ]
+
+    def __init__(self, debug=False, verbose=False):
+        self._defaults = self._parse_config()
+
+        if debug:
+            logging.basicConfig(level=logging.INFO)
+        elif verbose:
+            logging.basicConfig(level=logging.DEBUG)
+        else:
+            logging.basicConfig(level=logging.WARNING)
```

Comment:
I don't think it's appropriate to setup the logging from inside the module. That's a decision for a toplevel entry point.

Reply:
This was shuffled from when this was launched as a script but I'll move it.